src/harness/fourslash.ts

```diff
@@ -1818,6 +1818,21 @@ module FourSlash {
             }
         }
 
+        private verifyProjectInfo(expected: string[]) {
+            if (this.testType == FourSlashTestType.Server) {
+                let actual = (<ts.server.SessionClient>this.languageService).getProjectInfo(
+                    this.activeFile.fileName,
+                    /* needFileNameList */ true
+                    );
+                assert.equal(
+                    expected.join(","),
+                    actual.fileNameList.map( file => {
+                        return file.replace(this.basePath + "/", "")
+                        }).join(",")

```

Comment:
Is the list of files always guaranteed to be in a certain order?  As you just join the list and compare the string without sorting, this test may be fragile if the order ever changes.  May want to sort first.